library/alloc/src/rc.rs

```diff
@@ -396,13 +393,11 @@ impl<T> Rc<T> {
 
     /// Returns the inner value, if the `Rc` has exactly one strong reference.
     ///
-    /// Otherwise, an [`Err`][result] is returned with the same `Rc` that was
+    /// Otherwise, an [`Err`][Result] is returned with the same `Rc` that was
```

Comment:
This will resolve to the Err variant on the Result page :)
```suggestion
    /// Otherwise, an [`Err`] is returned with the same `Rc` that was
```